server/views/pages/goal/add-note/index.njk

```diff
@@ -17,8 +17,6 @@
     }) }}
   {% endif %}
 
-  {% include "../../../partials/prisonerBanner.njk" %}
```

Comment:
This is where the actual change starts. We are simply removing the include to the prisoner summary banner bar in all of the Goal related pages